lib/plugin/allure.js

```diff
@@ -244,6 +244,13 @@ module.exports = (config) => {
     }
   });
 
+  event.dispatcher.on(event.step.comment, (step) => {
+    reporter.startStep(`I say "${step.toString()}"`);
```

Comment:
```suggestion
    reporter.startStep(step.toString());
```

No need for `I.say` here, we need only the text.